backend/api/migrations/0033_urlcode_unique_case_insensitive_url_code.py

```
@@ -0,0 +1,18 @@
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0032_merge_20260611_1318'),
+    ]
+
+    operations = [
+        migrations.AddConstraint(
+            model_name='urlcode',
+            constraint=models.UniqueConstraint(django.db.models.functions.text.Lower('url_code'), name='unique_case_insensitive_url_code'),
+        ),
+    ]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)